### PR TITLE
Update CONFIGURATION.md

### PR DESCRIPTION
**Pull request by Leask**
https://github.com/ipfs/js-datastore-level/issues/85

### CODE REVIEW

**Review by vasco-santos — CHANGES_REQUESTED** [review]
Thanks for the PR @Leask 

The `interface-datastore` does not seem to mention that `.open()` will return the datastore. While this might be true today in some (or all) implementations of the interface, it can change in the future. So, can we change the added code in the docs to call open in a new line after creating the datastore?

**Comment by Leask** [comment]
@vasco-santos updated as your request.

File: doc/CONFIGURATION.md [hunk 1]
```diff
@@ -503,6 +503,9 @@ const MPLEX = require('libp2p-mplex')
 const { NOISE } = require('libp2p-noise')
 const LevelStore = require('datastore-level')
 
+const datastore = new LevelStore('path/to/store')
+const dsInstant = await datastore.open()
```
Comment (vasco-santos) [inline doc/CONFIGURATION.md:507]:
```suggestion
await datastore.open()
```

File: doc/CONFIGURATION.md [hunk 2]
```diff
@@ -511,7 +514,7 @@ const node = await Libp2p.create({
   },
   keychain: {
     pass: 'notsafepassword123456789',
-    datastore: new LevelStore('path/to/store')
+    datastore: dsInstant,
```
Comment (vasco-santos) [inline doc/CONFIGURATION.md:517]:
```suggestion
    datastore,
```

File: doc/CONFIGURATION.md [hunk 3]
```diff
@@ -673,16 +676,18 @@ const Libp2p = require('libp2p')
 const TCP = require('libp2p-tcp')
 const MPLEX = require('libp2p-mplex')
 const { NOISE } = require('libp2p-noise')
-
 const LevelStore = require('datastore-level')
 
+const datastore = new LevelStore('path/to/store')
+const dsInstant = await datastore.open()
```
Comment (vasco-santos) [inline doc/CONFIGURATION.md:682]:
```suggestion
await datastore.open()
```

File: doc/CONFIGURATION.md [hunk 4]
```diff
@@ -673,16 +676,18 @@ const Libp2p = require('libp2p')
...
+
 const node = await Libp2p.create({
   modules: {
     transport: [TCP],
     streamMuxer: [MPLEX],
     connEncryption: [NOISE]
   },
-  datastore: new LevelStore('path/to/store'),
+  datastore: dsInstant,
```
Comment (vasco-santos) [inline doc/CONFIGURATION.md:690]:
```suggestion
  datastore,
```

**Review by vasco-santos — CHANGES_REQUESTED** [review]

File: doc/CONFIGURATION.md (hunk 1 above)
Comment (vasco-santos) [reply doc/CONFIGURATION.md:507]: sorry if I was not clear enough, but I was meaning this way

**Comment by Leask** [comment]
I accepted all changes.

**Review by vasco-santos — APPROVED** [review]
Thanks @Leask

**Pull request merged**